interactions/api/http/interaction.py

```diff
@@ -216,17 +220,41 @@ async def get_all_application_command_permissions(
         )
 
     async def create_interaction_response(
-        self, token: str, application_id: int, data: dict
+        self,
+        token: str,
+        application_id: int,
+        data: dict,
+        files: Optional[List[File]] = MISSING,
```

Comment:
We don't use `MISSING` in the http files

Reply:
Applied.